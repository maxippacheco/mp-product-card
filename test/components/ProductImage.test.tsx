import React from 'react'
import renderer from 'react-test-renderer';
import { ProductCard, ProductImage } from '../../src/components';
import { product2 } from '../data/products';


describe('ProductImage', () => {
	test('debe de mostar el componente correctamente con el titulo personalizado', () => {

		const wrapper = renderer.create(
			<ProductImage 
				img='https://hola.jgg'
			/>
		)

		expect( wrapper.toJSON() ).toMatchSnapshot();
		

	});

	test('Debe de mostrar el componente con el nombre del product', () => {
		
		const wrapper = renderer.create(
			<ProductCard product={product2}>
				{
					() => (
						<ProductImage />
					)
				}
			</ProductCard>
		)

		expect( wrapper.toJSON() ).toMatchSnapshot();
		


	});
})






