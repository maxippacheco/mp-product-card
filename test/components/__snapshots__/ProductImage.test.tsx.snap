// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductImage Debe de mostrar el componente con el nombre del product 1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="Product"
    className="productImg undefined"
    src="./coffee-mug.png"
  />
</div>
`;

exports[`ProductImage debe de mostar el componente correctamente con el titulo personalizado 1`] = `
<img
  alt="Product"
  className="productImg undefined"
  src="https://hola.jgg"
/>
`;
